src/wp-includes/theme-previews.php

```diff
@@ -54,3 +54,27 @@ function wp_attach_theme_preview_middleware() {
 		'after'
 	);
 }
+
+/**
+ * Adds a nonce for the theme activation link.
+ * This is needed until we have a way to activate the theme via the REST API.
+ * See https://github.com/WordPress/gutenberg/pull/41836.
+ *
+ * @since 6.3.0
+ */
+function wp_block_theme_activate_nonce() {
+	$nonce_handle = 'switch-theme_' . wp_get_theme_preview_path();
+	?>
+<script type="text/javascript">
+	window.BLOCK_THEME_ACTIVATE_NONCE = '<?php echo wp_create_nonce( $nonce_handle ); ?>';
+</script>
```

Comment:
* Add WP prefix to global
* alignment nitpick

```suggestion
	<script type="text/javascript">
		window.WP_BLOCK_THEME_ACTIVATE_NONCE = '<?php echo wp_create_nonce( $nonce_handle ); ?>';
	</script>
```

Reply:
This requires an upstream change, PR submitted https://github.com/WordPress/gutenberg/pull/52398